flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -2099,13 +2099,13 @@ class ArrayExprLowering {
       }
     }
 
-    // Iteration space is created with outermost columns, innermost rows
```

Comment:
The comment looks correct to me, btw. Since FIR is column-major, the inner most loop should visit the row (left most index first), and so on until the outermost loop, which should visit the column (right most index).  That takes better advantage of cache locality.

Reply:
My bad, I understood column/rows as "going through data of a column in the outermost  loop" instead of "going through the column indexes in the outermost loop". I reverted this comment and fixed the new test ssa value naming.